src/main/java/kr/pickple/back/auth/config/property/KakaoOAuthConfig.java

```diff
@@ -0,0 +1,20 @@
+package kr.pickple.back.auth.config.property;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+@ConfigurationProperties(prefix = "oauth.kakao")
+public class KakaoOAuthConfig {
```

Comment:
```suggestion
public class KakaoOAuthProperties {
```
`@Configuration` 클래스들과 구분될 수 있도록 네이밍을 요렇게 해주는건 어떨까요?
(다른 property 클래스들도 동일)

Reply:
저도 되게 고민 많이했었는데 바꾸는게 더 명시적이고 좋을 것 같네요!
감사합니다!!

적용 커밋: [885dd46](https://github.com/Java-and-Script/pickple-back/pull/28/commits/885dd465e82bb15ee27f2908cb6e966c4ccb5dfa)